### DISCUSSION TITLE
optimize space? please

### DISCUSSION THREAD

**Discussion by 4lt3r3go** [Nevysha/Cozy-Nest#51] [Ideas]
this section doesnt really have much sense, it takes a lot of space over there and i dont think people use those 2 buttons very often, this could be moved somewere else or at least optimized
![image](https://github.com/Nevysha/Cozy-Nest/assets/44883288/66730c81-7307-4b9d-a915-bee00519ad5f)

**Comment by Nevysha** [comment]
>this section doesnt really have much sense, it takes a lot of space over there and i dont think people use those 2 buttons very often, this could be moved somewere else or at least optimized

Agree. That's not done on purpose but for a quick fix for something else (it was even worse before). I plan to work again on small fix like this after the next big update (with integrated image browser)
